### DISCUSSION TITLE
Merge CMILib and CMI Language Files

### DISCUSSION THREAD

**Discussion by AlternativeSoap** [Zrips/CMI#6829] [Feature Requests]
I think it would be better to merge CMI & CMILib Language files.

I just found out that most of the lines can be found in both files:
https://pastebin.com/xAEWiFFF

- It would then be easier to change language instead of changing most of one of the file, but it was the other one I should change... like really annoying.
- Yes, I use my time to change colors and text inside every language file.

**Comment by CloudeLecaw** [comment]
Cmi lib contains variabledms used in all zrips Plugins.
Cmilib was invented to share combined stuff between the plugins. So i dont think it will be merged aggain ;)

**Comment by mrfloris** [comment]
During the development of the library, some changes were made back and forth and kinda reverted. for best legacy support it's in a (what i would call) grace period. It won't surprise me that at some point the language file updates will include finalizing these steps and then people who've made customizations but not to both, will have to re-do a few things. 

Global phrases should end up in cmilib, this way all plugins benefit from them.
Phrases that are specific to a plugin, should remain in their own locale file. 

The changelogs on spigotmc have kept track of what changes were made, and semi-reverted at some point. 

I agree, they shouldn't be doubled-up. It creates confusion, but for legacy reasons it helped more than it caused confusion. Eventually I am sure it will smooth out
